provisioning_templates/PXELinux/pxelinux_global_default.erb

```diff
@@ -5,10 +5,11 @@ model: ProvisioningTemplate
 %>
 <%# Used to boot unknown hosts, do not associate or change the name. %>
 
-DEFAULT menu
+UI menu
```

Comment:
Have you omitted `c32` extension for a reason?

Reply:
I can't remember that I had a good reason for this, probably an oversight. I updated the template with the .c32 extension.